llvm_util/llvm2alive.cpp

```diff
@@ -332,9 +332,15 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
       if (!op)
         return error(i);
 
-      // TODO: support for struct indexing
-      if (I.getStructTypeOrNull())
-        return error(i);
+      if (auto structTy = I.getStructTypeOrNull()) {
+	auto const0 = llvm::ConstantInt::get(llvm::Type::getInt32Ty(i.getType()->getContext()), 0);
```

Comment:
it seems you've tabs here or smth

Reply:
yes, i will clean this up this weekend. It is not yet ready for review. Just hacked up something quick for an experiment.

Reply:
@nunoplopes okay. this is read for review now.